app/javascript/src/get-partial.js

```diff
@@ -34,7 +34,26 @@ function getPartial(event, element) {
 
   new FetchRails(url).get()
     .then(data => {
-      targetElement.dataset.loaded = "true"
       targetElement.innerHTML = data
     })
+    .then(() => {
+      if (_this.dataset.scrollOnLoad != "true") return
+
+      const hash = window.location.hash?.substring(1)
+      if (!hash) return
+
+      const scrollElement = document.getElementById(hash)
+      if (!scrollElement) return
+
+      const scrollOffset = scrollElement.getBoundingClientRect().top + document.documentElement.scrollTop
+
+      window.scrollTo({ top: scrollOffset - 70, behavior: "smooth" })
```

Comment:
Minor nit: it would nice to have a comment or something explaining this "magic" number of 70, although I could hazard a guess that this was due to the sticky header plus visual space.

Reply:
You're right, I got lazy and used a static number. I've replaced it with using the actual sticky tabs height so we don't have to think about this if we ever change those tabs or remove them altogether.